application/overlay/Jenkinsfile.twig

```diff
@@ -17,6 +17,11 @@ pipeline {
                 stage('go vet') { steps { sh 'ws go vet' } }
             }
         }
+        stage('Build for production') {
+            steps {
+                sh 'ws build-prod'
+            }
+        }
```

Comment:
the `Build` stage only builds a dev image (from the `golang` official image), which is huge but allows us to run `go test|vet|fmt|mod` in the `Test` stage, after that passes we want to rebuild the image ready for prod, which is just copying the built binary onto a scratch image before we then publish it